release/ray_release/scripts/ray_champagne.py

```diff
@@ -1,25 +1,48 @@
+import click
 import os
+from pathlib import Path
 
+import boto3
 from anyscale.sdk.anyscale_client.models.create_cluster_environment import (
     CreateClusterEnvironment,
 )
-
-from ray_release.aws import maybe_fetch_api_token
+from anyscale.sdk.anyscale_client.sdk import AnyscaleSDK
+from ray_release.bazel import bazel_runfile
 from ray_release.byod.build import build_champagne_image
 from ray_release.logger import logger
-from ray_release.util import get_anyscale_sdk
+from ray_release.configs.global_config import init_global_config
 
 CHAMPAGNE_IMAGE_TYPES = [
     # python_version, image_type, cluster_env_name
     ("py38", "cpu", "ray-champagne-cpu"),
     ("py38", "gpu", "ray-champagne-gpu"),
 ]
+ANYSCALE_SECRET_ARM = (
+    "arn:aws:secretsmanager:us-west-2:029272617770:secret:release-automation"
+)
+ANYSCALE_SECRETS = {
+    "anyscale-staging-token20221014164754935800000001-pfQunc": (
+        "https://console.anyscale-staging.com"
+    ),
+    "anyscale-demos-FaVACh": "https://console.anyscale.com",
+}
 
 
-def main() -> None:
+@click.option(
+    "--global-config",
+    default="oss_config.yaml",
+    type=click.Choice(
+        [x.name for x in (Path(__file__).parent.parent / "configs").glob("*.yaml")]
```

Comment:
nit: could you extract this into a global variable rather than inlining it here?

Reply:
of course